app/views/api/v2/authorizable.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @resource
+
+if @object.try(:class).try(:ancestors).to_a.include?(Authorizable)
+    node(:canEdit) { |resource| resource.authorized?(:"#{:edit}_#{resource.class.name.pluralize.downcase}") } 
```

Comment:
`#{:edit}` -> `edit` ;-)

But luckily we have a helper to get the permission name, `resource.permission_name('create')`.

Also please use snake case in the APIv2 for consistency.